src/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -149,11 +152,11 @@ private static boolean addToAvailableGames(final URI uri, final Map<String, URI>
     if (uri == null) {
       return false;
     }
-    InputStream input = null;
```

Comment:
General strategy of this change, find places where we create an InputStream, and above that line, have a 
`Optional<InputStream> inputStream = UrlStreams.openStream(uri);`
Then replace the initialization of the input stream with:
`try (InputStream input = inputStream.get()) {`

So in this case, we have:
`InputStream input = null;`

Turns into:
```
     Optional<InputStream> inputStream = UrlStreams.openStream(uri);
     if (inputStream.isPresent()) {
```